web/js/app.js

```diff
@@ -0,0 +1,9 @@
+require('jquery');
```

Comment:
The standard nowdays is to use import

Reply:
For now I'm staying at requiring the packages instead of importing them.